packages/react-tinacms-editor/src/core/index.tsx

```diff
@@ -44,34 +46,37 @@ export const Wysiwyg = styled(
     imageProps,
     ...styleProps
   }: WysiwygProps) => {
-    const { elRef: prosemirrorEl, editorView, translator } = useProsemirror(
-      input,
-      ALL_PLUGINS,
-      format,
-      imageProps
-    )
+    const [el, setEl] = useState<HTMLDivElement>()
+    const editorRef = useRef<HTMLDivElement | null>(null)
+
+    useEffect(() => {
+      if (editorRef.current) setEl(editorRef.current)
+    }, [editorRef])
 
     return (
       <WysiwygWrapper className="wysiwyg-wrapper">
         <link
           rel="stylesheet"
           href="https://codemirror.net/lib/codemirror.css"
         />
-        {editorView && (
+        <EditorStateProvider
+          input={input}
+          el={el}
```

Comment:
I think you could remove `el` and just use `editorRef`:

```suggestion
          el={editorRef.current}
```

You would probably need

Reply:
I need to trigger a re-render here so that provider gets value of `el`.

Reply:
Ahh yes I see. Could you add a comment explaining that above the `useState` for `el`?

Reply:
Sure I will do that